### PR TITLE
feat: Success token page

### PR DESCRIPTION
**Pull request by Tulun**
https://app.shortcut.com/uma-project/story/2322/create-new-success-token-page

Building of Success Token page.

@Tommy1231232 , can you work with Jesper on any specifics for mobile? like do we wanna hide the graph and such? There isn't a mobile design I think yet.

### CODE REVIEW

**Review by daywiss — COMMENTED** [review]
one tiny issue. theres a dash showing in the carausel for `put vesting agreements on-chain`
image

File: src/assets/scss/components/_products.scss [hunk 1]
```diff
@@ -387,6 +390,27 @@
...
+      flex-basis: 20%;
+    }
+    img {
+      // height: 100px;
+      margin-left: 3rem;
+      margin-top: -1rem;
+    }
+    p {
+      flex-basis: 75%;
+      font-family: "Halyard Display";
```
Comment (Gamaranto) [inline src/assets/scss/components/_products.scss:409]: i think this should be inherited and can be removed

**Review by Gamaranto — APPROVED** [review]
lgtm, just a small nit

File: src/assets/scss/components/_products.scss (hunk 1 above)
Comment (Tulun) [reply src/assets/scss/components/_products.scss]: Fair! done.

**Review by Gamaranto — APPROVED** [review]

**Review by daywiss — APPROVED** [review]

**Review by Gamaranto — APPROVED** [review]

**Pull request merged**